simplified_app.sh

```diff
@@ -40,6 +40,7 @@ git submodule update --init --recursive
 printf "$(git describe --tags)" > .version
 
 # Use the latest version of pip to install a virtual environment for the app.
+easy_install pip==9.0.3
 pip install -U --no-cache-dir pip setuptools
```

Comment:
I probably don't need to install setuptools with pip now that I'm installing python-setuptools above?